databricks/koalas/indexes.py

```diff
@@ -807,6 +807,27 @@ def name(self) -> str:
     def name(self, name: str) -> None:
         raise PandasNotImplementedError(class_name='pd.MultiIndex', property_name='name')
 
+    @property
+    def levshape(self) -> tuple:
+        """
+        A tuple with the length of each level.
+
+        Examples
+        --------
+        >>> midx = ks.MultiIndex.from_tuples([('a', 'x'), ('b', 'y'), ('c', 'z')])
+        >>> midx  # doctest: +SKIP
+        MultiIndex([('a', 'x'),
+                    ('b', 'y'),
+                    ('c', 'z')],
+                   )
+
+        >>> midx.levshape
+        (3,3)
```

Comment:
i think we skip this also.

Seems that it have different print out behavior depends on python version.

`(3,3)` for 3.6, but `(3, 3)` for 3.5 & 3.7

Reply:
Ok, it's a common issue in tuple.

Reply:
What's going on here?
I don't think the tuple string is not changed throughout 3.5 - 3.7.

```py
Python 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 16:30:03)
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> (3,3)
(3, 3)
```

```py
Python 3.6.9 |Anaconda, Inc.| (default, Jul 30 2019, 13:42:17)
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> (3,3)
(3, 3)
```

```py
Python 3.7.5 (default, Oct 25 2019, 10:52:18)
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> (3,3)
(3, 3)
```

Reply:
It should be `(3, 3)` and we should not skip it.

Reply:
ah, I see, the travis tests for the py3.6 environment are wrong.

Reply:
Submitted a PR #1107.

Reply:
Ah, that was a problem. Thanks for fixing @ueshin :+1: